WORKSPACE

```diff
@@ -104,6 +128,14 @@ filegroup(
     name = "code_proto",
     srcs = [ "google/rpc/code.proto" ],
 )
+
+filegroup(
+    name = "api_protos",
```

Comment:
As I'm going to add galley dependency which requires google/api/annotations.proto, I'm a bit worried about this name. Consider splitting this into `api_label_proto` and `api_metric_proto`.